<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSelcalCodeSelector</class>
 <widget class="QFrame" name="CSelcalCodeSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>151</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="hl_SelcalCodeSelector">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="cb_SelcalPairs1"/>
   </item>
   <item>
    <widget class="QComboBox" name="cb_SelcalPairs2"/>
   </item>
   <item>
    <widget class="BlackGui::CTickLabel" name="lblp_ValidCodeIcon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../blackmisc/blackmisc.qrc">:/diagona/icons/diagona/icons/tick-button.png</pixmap>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::CTickLabel</class>
   <extends>QLabel</extends>
   <header>blackgui/ticklabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../blackmisc/blackmisc.qrc"/>
 </resources>
 <connections/>
</ui>
